src/other/broadcasting.jl

```diff
@@ -0,0 +1,63 @@
+struct LazyNewColDataFrame
+    df::DataFrame
+    col::Symbol
+end
+
+# we allow LazyNewColDataFrame only for data frames with some columns
+Base.axes(x::LazyNewColDataFrame) = axes(x.df[1])
+
+Base.Broadcast.broadcastable(df::AbstractDataFrame) = Matrix(df)
+
+Base.maybeview(df::AbstractDataFrame, idxs...) = view(df, idxs...)
+
+function Base.maybeview(df::AbstractDataFrame, idxs)
+    if ncol(df) == 0
+        throw(ArgumentError("Broadcasting into a data frame with no columns is not allowed"))
+    end
+    if idxs isa Symbol
+        if !haskey(df, idxs)
+            if !(df isa DataFrame)
+                throw(ArgumentError("column $idxs not found"))
```

Comment:
Can we print suggestions of similar column names as in `getindex`?

Reply:
OK - I will add this